spec/openapi.yaml

```diff
@@ -775,6 +779,67 @@ paths:
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+
+#
+# Peoples API
+#
+
+  /people-api/v2/users:
+    post:
+      summary: Get list of user information
+      tags:
+      - Peoples
+      description: >
+        People is user of the system who process, translate context
+
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                userUids:
+                  description: >-
+                    List of user uids.
+                  example:
+                    - "ac48ht4qy"
+					
+                  items:
+                    type: string
+                  type: array
+                excludeProfileImage:
+                  description: >-
+                    Exclude user profile image. It is optimization to avoid fetching user's profile image
```

Comment:
As a performance optimization, exclude the user's profile image.

Reply:
Done